src/pages/workspace/distanceRates/CreateDistanceRatePage.tsx

```diff
@@ -32,8 +33,8 @@ function CreateDistanceRatePage({route}: CreateDistanceRatePageProps) {
     const policyID = route.params.policyID;
     const policy = usePolicy(policyID);
     const currency = policy?.outputCurrency ?? CONST.CURRENCY.USD;
-    const customUnits = policy?.customUnits ?? {};
-    const customUnitID = customUnits[Object.keys(customUnits)[0]]?.customUnitID ?? '';
+    const distanceRateCustomUnit = getDistanceRateCustomUnit(policy);
```

Comment:
```suggestion
    const customUnit = getDistanceRateCustomUnit(policy);
```

Reply:
@shubham1206agra Can you revert all the variable names changes to customUnit. It's creating extra change and inconsistency in variable names